plugins/@grouparoo/sendgrid/__tests__/export/export-profile.ts

```diff
@@ -398,6 +455,36 @@ describe("sendgrid/exportProfile", () => {
     expect(oldUser).toBe(null);
   });
 
+  test("it can change the email address and orphan the old user if sync mode is not deleting", async () => {
+    // sendgrid requires deleting the old user on FK change
+    await runExport({
+      syncOperations: { create: true, update: true, delete: false },
+      oldProfileProperties: {
+        email: alternativeEmail,
+      },
+      newProfileProperties: {
+        email,
+      },
+      oldGroups: [listOne, listTwo, listThree],
+      newGroups: [listOne, listTwo, listThree],
+      toDelete: false,
+    });
+
+    await indexContacts(newNock);
+
+    // old user still there
+    const oldUser = await apiClient.getUser(alternativeEmail);
+    expect(oldUser).not.toBe(null);
+    expect(oldUser.email).toBe(alternativeEmail);
+    expect(oldUser["list_ids"]).toHaveLength(0); // but has been removed from lists
```

Comment:
The user with alternativeEmail did not have any lists to start with. You may want to add these groups to the previous text where this user is created to ensure it has the lists in the first place and it's now being deleted. My initial impression is that this would not happen.